velox/functions/lib/string/StringImpl.h

```diff
@@ -197,38 +197,110 @@ std::vector<int32_t> stringToCodePoints(const T& inputString) {
   return codePoints;
 }
 
-/// Returns the starting position in characters of the Nth instance(counting
-/// from the left if lpos==true and from the end otherwise) of the substring in
-/// string. Positions start with 1. If not found, 0 is returned. If subString is
-/// empty result is 1.
+/// Returns the starting position in characters of the Nth instance of the
+/// substring in string after 'startPosition'. Positions start with 1. If not
+/// found, 0 is returned. If subString is empty result is 1.
+/// @tparam lpos If true, counting from the left and from the end otherwise.
+/// @param instance The instance of the substring to find in string.
```

Comment:
Would you clarify whether this is 0- or 1-based?

Reply:
Updated. It is 1-based.